server/lib/routes/get-metrics-flow.js

```diff
@@ -0,0 +1,56 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const flowMetrics = require('../flow-metrics');
+const logFlowEvent = require('../flow-event').logFlowEvent;
+const logger = require('../logging/log')('server.get-metrics-flow');
+
+module.exports = function (config) {
+  const FLOW_ID_KEY = config.get('flow_id_key');
+  const FLOW_EVENT_NAME = 'flow.begin';
```

Comment:
We'll also need to remove `app/scripts/views/mixins/flow-begin-mixin.js` from the client-side of course. Probably you already realised that, just making sure.

Any places that were mixing it in can mix in `flow-events-mixin.js` instead now, I think, and everything should just magically work.

Reply:
hmmm, We cannot just remove flow-begin from client-side because there are still iframe versions of the flow. So we either not record a begin if there was a query flow sent or some other solution

Reply:
Oh good point, lol. Yeah, maybe some condition to the begin mixin so it doesn't emit the event if the query params are set?